projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget.model.ts

```diff
@@ -185,13 +187,17 @@ export class TableWidgetModel {
   }
 
   public getColumns(scope?: string): Observable<SpecificationBackedTableColumnDef[]> {
-    const modelColumns = this.columns.map(column => column.asTableColumnDef());
+    const modelColumns: Observable<SpecificationBackedTableColumnDef[]> = forkJoinSafeEmpty(
+      this.columns.map(column => column.asTableColumnDef(scope))
+    );
 
     if (scope === undefined || !this.fetchEditableColumns) {
-      return of(modelColumns);
+      return modelColumns;
     }
 
-    return this.tableWidgetColumnsService.fetchColumn(scope, modelColumns);
+    return modelColumns.pipe(
+      switchMap(modelColumns => this.tableWidgetColumnsService.fetchColumn(scope, modelColumns))
```

Comment:
Isn't this shadowing the observable variable? Can we rename it to `modelColumns$` and `modelColumns` respectively?